### PR TITLE
Feature/contact merge audit

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

This PR introduces the `stg_hubspot__contact_audit_merge` model to the source package which will be leveraged in the downstream models to resolve and contact merge discrepancies if needed.

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [X] No  (please provide explanation how the change is non breaking below.)

This PR only introduces a new staging model that is disabled by default. No breaking change.

**Is this PR in response to a previously created Issue**

- [X] Yes, Issue [46](https://github.com/fivetran/dbt_hubspot/issues/46)
- [ ] No 

**How did you test the PR changes?** 

 
- [X] CircleCi  
- [ ] Other (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 
- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🌵 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: models/src_hubspot.yml [hunk 1]
```diff
@@ -150,6 +150,30 @@ sources:
...
+          - name: canonical_vid
+            description: The contact ID of the contact which the vid_to_merge contact was merged into.
+          - name: contact_id
+            description: The ID of the contact.
+          - name: entity_id
+            description: The ID of the related entity.
+          - name: first_name
+            description: The contacts first name.
+          - name: last_name
+            description: The contacts last name.
```
Comment (fivetran-jamie) [inline models/src_hubspot.yml:167]:
```suggestion
          - name: first_name
            description: The contact's first name.
          - name: last_name
            description: The contact's last name.
```

File: models/stg_hubspot__contact.yml [hunk 2]
```diff
@@ -93,6 +93,30 @@ models:
...
+      - name: canonical_vid
+        description: The contact ID of the contact which the vid_to_merge contact was merged into.
+      - name: contact_id
+        description: The ID of the contact.
+      - name: entity_id
+        description: The ID of the related entity.
+      - name: first_name
+        description: The contacts first name.
+      - name: last_name
+        description: The contacts last name.
```
Comment (fivetran-jamie) [inline models/stg_hubspot__contact.yml:110]:
```suggestion
      - name: first_name
        description: The contact's first name.
      - name: last_name
        description: The contact's last name.
```

**Review by fivetran-jamie — APPROVED** [review]
lgtm! just a couple of tiny doc suggestions

File: models/src_hubspot.yml (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply models/src_hubspot.yml:167]: 🤦 great catch!

File: models/stg_hubspot__contact.yml (hunk 2 above)
Comment (fivetran-joemarkiewicz) [reply models/stg_hubspot__contact.yml:110]: 🤦 Another great catch

**Pull request merged**